main.tf

```diff
@@ -141,7 +141,6 @@ module "takeover_role" {
   project                  = var.project
   security_audit_role_name = var.security_audit_role_name
   kms_arn                  = module.kms.kms_arn
-  takeover                 = var.takeover
```

Comment:
How come var.takeover is removed? Is the var no longer needed?

Reply:
Well, sounds easy when you don't even call it. See main:
```
module "takeover_role" {
  count  = var.takeover ? 1 : 0
  source = "./modules/iam"

  project                  = var.project
  security_audit_role_name = var.security_audit_role_name
  kms_arn                  = module.kms.kms_arn
  policy                   = "takeover"
  permissions_boundary_arn = var.permissions_boundary_arn
  environment              = local.env
}
```

Reply:
Anyway, it was broken. Needed to pass the variable set.

Reply:
Reverted.